concent_api/requirements.lock

```diff
@@ -1,3 +1,24 @@
+asn1crypto==0.23.0
+bitcoin==1.1.42
+cbor2==3.0.4
+cffi==1.11.2
+coincurve==6.0.0
 Django==1.11.7
+freezegun==0.3.9
```

Comment:
`golem-messages` does not depend on `freezegun`,  `python-dateutil` and `six`. These packages should not be listed here. Please regenerate `requirements.lock` on a clean virtualenv to make sure you have no stray packages here.